airflow/www/app.py

```diff
@@ -75,6 +75,9 @@ def create_app(config=None, testing=False):
 
     flask_app.config["PERMANENT_SESSION_LIFETIME"] = timedelta(minutes=settings.get_session_lifetime_config())
 
+    limit_in_mb = 1
```

Comment:
IMHO, this value should be configurable by Airflow webserver configurations.

Reply:
@hussein-awala 
As @dirrao also mentioned, indeed that makes sense. I'll start working on this first.

Reply:
@hussein-awala @dirrao 
I made the value configurable by placing it in the webserver section of airflow.cfg.
Please check it when you have time.